### PR TITLE
more useful and human-readable feedback

### PR DESCRIPTION
**Pull request by natechols**

### CODE REVIEW

File: smrt-server-tools/src/main/scala/com/pacbio/secondary/smrttools/client/ServiceAccessLayer.scala [hunk 1]
```diff
@@ -254,19 +254,24 @@ class ServiceAccessLayer(val baseUrl: URL)(implicit actorSystem: ActorSystem) {
     Get(toJobResourceUrl(JobTypes.PB_PIPE, jobId, ServiceResourceTypes.ENTRY_POINTS))
   }
 
-  private def getJobDataStore(jobType: String, jobId: Int) : Future[PacBioDataStore] = getDataStorePipeline {
+  // FIXME I think this is unmarshalling incorrectly
```
Comment (natechols) [inline smrt-server-tools/src/main/scala/com/pacbio/secondary/smrttools/client/ServiceAccessLayer.scala]: The problem here is currently: "Object is missing required member 'uniqueId'".  But using PacBioDataStore didn't work either - it complaned about "createdAt" (and that seems inappropriate anyway).

Comment (mpkocher) [inline smrt-server-tools/src/main/scala/com/pacbio/secondary/smrttools/client/ServiceAccessLayer.scala]: So the call is something like this?

http://smrtlink-beta:8081/secondary-analysis/job-manager/jobs/pbsmrtpipe/6/datastore

Comment (mpkocher) [inline smrt-server-tools/src/main/scala/com/pacbio/secondary/smrttools/client/ServiceAccessLayer.scala]: I believe it needs to be DataStoreServiceFile.

By convention, there's often a non-service data model, then a wrapped service data model. For example the job options and the datastore file follow this pattern.

Comment (natechols) [inline smrt-server-tools/src/main/scala/com/pacbio/secondary/smrttools/client/ServiceAccessLayer.scala]: That worked.

**Pull request merged**